cmd/protoc-gen-openapi/main.go

```diff
@@ -35,6 +35,7 @@ func main() {
 		EnumType:        flags.String("enum_type", "integer", `type for enum serialization. Use "string" for string-based serialization`),
 		CircularDepth:   flags.Int("depth", 2, "depth of recursion for circular messages"),
 		DefaultResponse: flags.Bool("default_response", true, `add default response. If "true", automatically adds a default response to operations which use the google.rpc.Status message. Useful if you use envoy or grpc-gateway to transcode as they use this type for their default error responses.`),
+		OneofType:       flags.String("oneof_type", "ignore", `type for oneof wrap. Use "wrapped" to wrap fields into oneof's name property`),
```

Comment:
Does "ignore" mean that we omit the oneof fields or that we flatten them (as done in the tests at the top of the PR)? If we're flattening, then I think "flattened" (or just "flat") might be a better option value.

Also, for general conciseness, how about making these values "flat" and "wrap"? I think the option name could just be "oneof".

This is a bit different from "enum_type", which is specifying an actual type (integer or string), so I don't think the "_type" suffix is necessary or clarifying here.

Reply:
great, I will adjust this option. `oneof` with available value `flat` and `wrap`, much more understandable